pkg/utils/gwlog/gwlog_test.go

```diff
@@ -0,0 +1 @@
+package gwlog
```

Comment:
What's the purpose of this file?

Reply:
leftover after renaming log to gwlog, there was empty test file. will remove